ui-app/client/src/components/SmallCarousel/smallCarouselProperties.ts

```diff
@@ -0,0 +1,362 @@
+	ComponentPropertyDefinition,
+	ComponentPropertyEditor,
+	ComponentPropertyGroup,
+} from '../../types/common';
+import { COMMON_COMPONENT_PROPERTIES, COMPONENT_STYLE_GROUP_PROPERTIES } from '../util/properties';
+
+const propertiesDefinition: Array<ComponentPropertyDefinition> = [
+	{
+		name: 'showArrowButtons',
+		schema: SCHEMA_BOOL_COMP_PROP,
+		displayName: 'Left and right Buttons',
+		description: 'Left and right arrow buttons to control carousel',
+		group: ComponentPropertyGroup.BASIC,
+		defaultValue: true,
+	},
+	{
+		name: 'showDotsButtons',
+		schema: SCHEMA_BOOL_COMP_PROP,
+		displayName: 'Dot buttons in carousel',
+		description: `Dot buttons that will show on bottom of the carousel.`,
+		group: ComponentPropertyGroup.BASIC,
+		defaultValue: true,
+	},
+	{
+		name: 'slideSpeed',
+		schema: SCHEMA_NUM_COMP_PROP,
+		displayName: 'Carousel slide speed',
+		description: `Adjust the slide speed by giving the time in milliseconds.`,
+		group: ComponentPropertyGroup.BASIC,
+		defaultValue: 2000,
+	},
+	{
+		name: 'animationDuration',
+		schema: SCHEMA_NUM_COMP_PROP,
+		displayName: 'Animation Duration',
+		description: 'How long the animation should last.',
+		group: ComponentPropertyGroup.ADVANCED,
+		defaultValue: 2000,
+	},
+	{
+		name: 'easing',
+		schema: SCHEMA_STRING_COMP_PROP,
+		displayName: 'Easing',
+		description: 'Easing',
+		editor: ComponentPropertyEditor.ENUM,
+		group: ComponentPropertyGroup.ADVANCED,
+		defaultValue: 'ease',
+		enumValues: [
+			{ name: 'linear', displayName: 'Linear', description: 'Linear' },
+			{ name: 'ease', displayName: 'Ease', description: 'Ease' },
+			{ name: 'ease-in', displayName: 'Ease In', description: 'Ease In' },
+			{ name: 'ease-out', displayName: 'Ease Out', description: 'Ease Out' },
+			{ name: 'ease-in-out', displayName: 'Ease In Out', description: 'Ease In Out' },
+		],
+	},
+	{
+		name: 'autoPlay',
+		schema: SCHEMA_BOOL_COMP_PROP,
+		displayName: 'Auto Play',
+		description: 'Play automatically without interaction',
+		group: ComponentPropertyGroup.BASIC,
+		defaultValue: true,
+	},
+	{
+		name: 'dotsButtonType',
+		schema: SCHEMA_STRING_COMP_PROP,
+		displayName: 'dotsButtonType',
+		description: 'Types of Dot buttons',
+		editor: ComponentPropertyEditor.ENUM,
+		defaultValue: 'circle',
+		group: ComponentPropertyGroup.BASIC,
+		enumValues: [
+			{ name: 'circle', displayName: 'dots', description: 'dots' },
+			{ name: 'square', displayName: 'square', description: 'square buttons' },
+			{ name: 'minus', displayName: 'dash', description: 'dash icon' },
+			{ name: 'none', displayName: 'none', description: 'none' },
+		],
+	},
+	{
+		name: 'dotsButtonIconType',
+		schema: SCHEMA_STRING_COMP_PROP,
+		displayName: 'dotsButtonType',
+		description: 'Position of Dot buttons',
+		editor: ComponentPropertyEditor.ENUM,
+		group: ComponentPropertyGroup.BASIC,
+		defaultValue: 'regular',
+		enumValues: [
+			{ name: 'regular', displayName: 'regular', description: 'hallo icon' },
```

Comment:
Same capitalize first letter in displayName and description. Correct the spelling of hallow.